### PR TITLE
Feature/add order properties

### PR DESCRIPTION
**Pull request by SpiderDan98**
Add product gender
Add product categories as string

### CODE REVIEW

File: src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+     *
+     * @return string|null
+     */
+    public function getGender(JellyfishOrderItemTransfer $jellyfishOrderItemTransfer): ?string
+    {
+        $productConcreteTransfer = $this->productFacade->getProductConcrete($jellyfishOrderItemTransfer->getSku());
+
+        $attributes = $this->productFacade->getCombinedConcreteAttributes($productConcreteTransfer);
+
+        return $attributes ? $attributes["gender"] ?? null : null;
```
Comment (julianhyatt) [inline src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php]: single quote

File: src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * @param \Generated\Shared\Transfer\CategoryCollectionTransfer $categoryCollectionTransfer
+     *
+     * @return string|null
+     */
+    protected function mapCategoryCollectionToString(CategoryCollectionTransfer $categoryCollectionTransfer): ?string
+    {
+        $categories = $categoryCollectionTransfer->getCategories();
+        $categoryString = "";
```
Comment (julianhyatt) [inline src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php]: single quote

File: src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        if ($categories->count() === 0) {
+            return null;
+        }
+
+        foreach ($categories as $key => $category) {
+            if ($key === 0) {
+                $categoryString .= $category->getName();
+                continue;
+            }
+            $categoryString .= $category->getName() . ", ";
```
Comment (julianhyatt) [inline src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php]: single quote

File: src/FondOfSpryker/Zed/JellyfishCrossEngage/JellyfishCrossEngageDependencyProvider.php [hunk 4]
```diff
@@ -2,8 +2,79 @@
...
 
+use FondOfSpryker\Zed\JellyfishCrossEngage\Dependency\Client\JellyfishCrossEngageToLocaleFacadeBridge;
+use FondOfSpryker\Zed\JellyfishCrossEngage\Dependency\Client\JellyfishCrossEngageToProductCategoryFacadeBridge;
+use FondOfSpryker\Zed\JellyfishCrossEngage\Dependency\Client\JellyfishCrossEngageToProductFacadeBridge;
 use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
 
 class JellyfishCrossEngageDependencyProvider extends AbstractBundleDependencyProvider
 {
+    public const PRODUCT_FACADE = "PRODUCT_FACADE";
```
Comment (julianhyatt) [inline src/FondOfSpryker/Zed/JellyfishCrossEngage/JellyfishCrossEngageDependencyProvider.php]: single quotes

File: tests/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngageFacadeTest.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+     * @var \ArrayObject|\string[][]
+     */
+    protected $categories;
+
+    /**
+     * @return void
+     */
+    protected function _before(): void
+    {
+        $this->gender = "male";
```
Comment (julianhyatt) [inline tests/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngageFacadeTest.php]: single quotes

**Review by julianhyatt — COMMENTED** [review]
check am besten mal alles auf single quotes

File: src/FondOfSpryker/Shared/JellyfishCrossEngage/JellyfishCrossEngageConstants.php [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace FondOfSpryker\Shared\JellyfishCrossEngage;
+
+interface JellyfishCrossEngageConstants
+{
+    public const DEFAULT_LOCALE_NAME = "JELLYFISH_CROSS_ENGAGE:DEFAULT_LOCALE_NAME";
```
Comment (julianhyatt) [inline src/FondOfSpryker/Shared/JellyfishCrossEngage/JellyfishCrossEngageConstants.php]: Quotes :)

**Review by julianhyatt — APPROVED** [review]

**Review by sourcecube — APPROVED** [review]

File: src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php (hunk 1 above)
Comment (daniel-rose) [reply src/FondOfSpryker/Zed/JellyfishCrossEngage/Business/JellyfishCrossEngage/JellyfishCrossEngageReader.php]: finde verschachtelte inline ifs nicht lesbar...

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**